internal/pkg/levee/testdata/src/example.com/tests/panic/tests.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package panic
```

Comment:
I think these cases are good, but I would like to also see the following:

* Panicking on a non-tainted, non-sanitized value does not produce a report. Make sure there is a Source in scope so that there is actually a possibility of failure. This would be intended as a sanity check; I think it's good to have given that we don't have many `panic`s in our tests.
* When `AllowPanicOnTaintedValues` is set to `true`, panicking on a tainted value does not produce a report.

Reply:
I added the panicking on a non-tainted, non-sanitized value does not produce a report case. For the AllowPanicOnTaintedValues case I am not sure if the current testing framework lets us test that easily. I think Ill open an issue for that and fix that later. In the meantime I have manually verified that setting AllowPanicOnTaintedValues to true does not produce reports for panic on tainted values.

Reply:
This is behavior that is being added by this PR, and as such I believe that it should be fully covered by automated tests in this PR.

> I am not sure the current testing framework lets us test that easily.

It does. Our `config` package allows us to set the path to the config, and `analysistest` allows the package to test to be specified.

Reply:
Done.